### ISSUE TITLE
"semi" rule broken with ESLint 4.7.0

### ISSUE THREAD

**Issue by coderpatsy** [Microsoft/vscode-eslint#300]
VSCode version: 1.16.1
ESLint version: 4.7.0
ESLint plugin version: 1.3.1

.eslintrc.json config:
```js
{
    "rules": {
        "semi": "error"
    }
}
```
JavaScript:
```js
var test = false
```

Expected behavior:
1 error for missing semicolon

Actual behavior:
0 errors

The rule works fine in the plugin with ESLint 4.6.1, and ESLint command line notices the error in both 4.6.1 and 4.7.0.

**Comment by dbaeumer** [comment]
That still works for me:

<img width="869" alt="capture" src="https://user-images.githubusercontent.com/1931590/30583040-181b1f10-9d26-11e7-9fc4-8e59781e0dd6.png">

Do you have a GitHub repository I can clone to reproduce what I am seeing?

**Comment by dbaeumer** [comment]
I used ESLint 4.7.1 (latest). May be it was broken in 4.7.0 :-)

**Comment by coderpatsy** [comment]
Folder's just the config and a test.js as above.

However the plugin seems to be working fully again. Installing 4.7.1 didn't fix it, so I turned off all my user settings in VSCode. After a forced restart, it's working again. It's still working even with my user settings back on, and with 4.7.0.

I should say, I did close all VSCode windows between installs of ESLint, so I don't know what was different there. Some phantom unreproducable bug in VSCode itself or something? Ah well it's fixed.
